solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/Feature.java

```diff
@@ -207,8 +189,16 @@ public String getName() {
       return Feature.this.name;
     }
 
+    public void setNorm(Normalizer norm) {
+      this.norm = norm;
```

Comment:
Does the Weight need to have set function and a private norm variable?  Can the getNorm just return Feature.this.norm; like it did before?